src/ui/SmartSnippet/SmartSnippet.ts

```diff
@@ -60,19 +68,45 @@ export class SmartSnippet extends Component {
   }
 
   public createDom() {
-    this.buildShadow();
-    this.buildSourceContainer();
+    this.element.appendChild(this.buildAnswerContainer());
+    this.element.appendChild(
+      new UserFeedbackBanner(
+        isUseful =>
+          this.usageAnalytics.logCustomEvent<IAnalyticsNoMeta>(
+            isUseful ? analyticsActionCauseList.likeSmartSnippet : analyticsActionCauseList.dislikeSmartSnippet,
+            {},
+            this.element,
+            this.lastRenderedResult
+          ),
+        explaination => console.info(`Explaination: ${explaination}`)
```

Comment:
Currently, the `Explain why` button doesn't open a modal like it does in the design. The primary reason being that I didn't want to make this PR even larger, and that I'm not sure whether or not we actually want to add such a modal.

What do you think about it?

Reply:
I'd remove the property for now instead of having it do "nothing".

Also it should be `explanation`

Reply:
Woops, my bad! And I'll remove it.